src/Microsoft.DotNet.XHarness.CLI/Program.cs

```diff
@@ -85,5 +94,26 @@ public static CommandSet GetXHarnessCommandSet()
 
             return commandSet;
         }
+
+        /// <summary>
+        /// Returns true when the command outputs data suitable for parsing and we should keep the output clean.
+        /// </summary>
+        private static bool IsOutputSensitive(string[] args)
+        {
+            if (args.Length < 2 || args.Contains("--help") || args.Contains("-h"))
+            {
+                return false;
+            }
+
+            switch (args[0])
+            {
+                case "ios":
+                case "apple":
+                case "android":

```

Comment:
@greenEkatherine this will change the behaviour of the `android device` command so that the output is easier to parse. But we should probably let the users (Fan Yang) know about this.

Do you want this change for Android?

Reply:
yes, I think it would be great, I will simplify the command in unit tests as well. Do we have a channel for such announcements or should I notify her directly?

Reply:
I don't think they are aware of the Teams channel we created for mono/Xamarin, so maybe we would need to reach out to her. I would merge this then so that you can use this change in your integration tests?